src/rrdp-iostat/rrdp_iostat.ml

```diff
@@ -264,12 +273,12 @@ let exec_tap_ctl () =
             D.debug "no knowledge about pid %d; ignoring" pid
         ) disappeared_pids;
 
-      if unmapped_vdi_pids <> [] then begin
-        let pids_to_string pids = pids |> List.map (fun (pid, (_, (_, vdi))) -> Printf.sprintf "%d (%s)" pid vdi) |> String.concat "; " in
-        D.info "There are new tapdisk processes: [%s]; updating VDI-to-VM map..." (pids_to_string unmapped_vdi_pids);
```

Comment:
I think we missed updating this when we did the SMAPIv3 code, there are now also qemu processes, not just tapdisk processes, and if in the future we move to using qemu more  then we should fix that. However that would be out of scope for this XSI.

Reply:
okay, i will assume you have made a note of this somewhere